spec/build_command_generator_spec.rb

```diff
@@ -7,7 +9,11 @@
     end
 
     it "supports additional parameters" do
-      options = { project: "./example/standard/Example.xcodeproj", sdk: "9.0" }
+      xcargs_hash = { DEBUG: "1", BUNDLE_NAME: "Example App" }
```

Comment:
Would it make sense to allow the user to also allow passing a hash and `gym` automatically converts it into a string (like you did in this test)